spec/git-repository-provider-spec.js

```diff
@@ -13,6 +13,14 @@ describe('GitRepositoryProvider', () => {
     provider = new GitRepositoryProvider(atom.project, atom.config, atom.confirm)
   })
 
+  afterEach(() => {
```

Comment:
Why was this added?

Reply:
Because repositories are disposable and none of the tests perform cleanup, and each `describe` gets a new provider (losing the handle to the previous and making the provider subject to the GC which may not run before the tests finish). Just trying to make the test environment as clean as possible without memory leaks =]